src/themes/money/theme.json

```diff
@@ -266,6 +266,8 @@
   },
 
   "articleIntro": {
+    "contentCols": 12,
```

Comment:
Why do we keep redefining `contentCols`? Surely this should use the grid defaults...